azure-storage-file/azure/storage/file/_deserialization.py

```diff
@@ -197,6 +200,71 @@ def _convert_xml_to_directories_and_files(response):
     return entries
 
 
+def _convert_xml_to_handles(response):
+    """
+    <?xml version="1.0" encoding="utf-8"?>
+    <EnumerationResults>
+      <HandleList>
+        <Handle>
+          <HandleId>handle-id</HandleId>
+          <Path>file-or-directory-name-including-full-path</Path>
+          <FileId>file-id</FileId>
+          <ParentId>parent-file-id</ParentId>
+          <SessionId>session-id</SessionId>
+          <ClientIp>client-ip</ClientIp>
+          <OpenTime>opened-time</OpenTime>
+          <LastReconnectTime>lastreconnect-time</LastReconnectTime>
+        </Handle>
+        ...
+      </HandleList>
```

Comment:
Minor - this doesn't match the actual string we're looking for.  I think the code is correct and the docs are wrong, though.

Reply:
Oh yes, the spec was outdated.